tests/dummy/app/modifiers/draw-stacked-chart.js

```diff
@@ -15,7 +15,7 @@ import Modifier from 'ember-modifier';
 const musicFormats = Object.keys(COLOR_PALETTE);
 const paletteColors = Object.values(COLOR_PALETTE);
 
-export default class DrawChartModifier extends Modifier {
+export default class DrawStackedChartModifier extends Modifier {
```

Comment:
I gave up on installing `@types/d3-*` packages. The [sample D3 code](https://observablehq.com/@mbostock/revenue-by-music-format-1973-2018) has too many TypeScript errors.